src/Control/Lens/Tuple.hs

```diff
@@ -110,6 +111,9 @@ class Field1 s t a b | s -> a, t -> b, s b -> t, t a -> s where
 instance Field1 (Identity a) (Identity b) a b where
   _1 f (Identity a) = Identity <$> f a
 
+instance Field1 (Unit a) (Unit b) a b where
+  _1 f (Unit a) = Unit <$> f a
```

Comment:
Or should this be lazy???? I think *all* the tuple instances should be strict, but I know others disagree...

Reply:
I'd vote to keep it the way you've written it, as it's consistent with the other `Field1` instances that are currently defined. If others feel strongly about changing this convention, then I think it would be best to leave that to a separate issue.

Reply:
Other `Field1` insances are inconsistent. See `Pair f g a` and `(a, b)`.

Reply:
Oops, I'm not sure what I was thinking when I wrote https://github.com/ekmett/lens/pull/878/files#r306307359, as the tuple instances are indeed implemented using irrefutable pattern matches. Hrm. I don't personally have a strong opinion on this, but for the sake of avoiding this PR from being bogged down by bikeshedding, it might be wise to stick to the current convention for now and raise a separate issue about how to reconcile the strictness of the instances for tuples and `Product`.

Reply:
We have `_1'` for when you want to reduce leaks